index.html

```diff
@@ -3,18 +3,18 @@
 <div id="FirebugChannel" style="display: none;">FB_toggle</div>

```

Comment:
Why does it here?
Removes or replaces, please.

Take a look, please.
`http://htmlbook.ru/samhtml/struktura-html-koda`

Reply:
done